tsdef/types.d.ts

```diff
@@ -148,7 +148,6 @@ export interface LocalTrackPublishOptions {
 }
 
 export interface MediaStreamTrackPublishOptions {
-  name?: string;
```

Comment:
@PikaJoyce I think we should keep this value. It's in the docs. Plus, without it, `MediaStreamTrackPublishOptions` and `LocalTrackPublishOptions` would be identical. 

What do you think?

Reply:
@timmydoza, I removed this because the documentation for [MediaStreamTrackPublishOptions](https://media.twiliocdn.com/sdk/js/video/releases/2.13.0/docs/global.html#MediaStreamTrackPublishOptions) does not have a name property. Which is why I removed it. Is the documentation incorrect? If it is, I can change the documentation for `MediaStreamTrackPublishOptions` and put the name property back in. 

cc: @manjeshbhargav

Reply:
@timmydoza @PikaJoyce Yes, `MediaStreamTrackPublishOptions` should have the `name` property. Please correct the JS documentation to include it.

Reply:
@PikaJoyce - Looking closer, there isn't a `name` property on the `MediaStreamTrackPublishOptions` type itself, but there is a `name` property on the `LocalTrackOptions` type that `MediaStreamTrackPublishOptions` inherits from. So maybe something like this?

```
export interface MediaStreamTrackPublishOptions extends LocalTrackOptions {
    priority?: Track.Priority;
}
```

This looks like its consistent with the docs: https://media.twiliocdn.com/sdk/js/video/releases/2.13.0/docs/global.html#MediaStreamTrackPublishOptions

Reply:
@PikaJoyce Looks like you don't need to create the documentation ticket after all! 🎉

Reply:
@timmydoza Oh yay! Thanks for finding that. For some odd reason, I kept reading that `MediaStreamTrackPublishOptions` inherits from `LocalTrackPublishOptions` and not `LocalTrackOptions`. That was  my mistake, I'm sorry. I'll revert the change and add in your suggestion :)

Reply:
Also it looks like I didn't even commit the JSDoc change so there is conveniently nothing for me to change on that end!